internal/db/resources_test.go

```diff
@@ -0,0 +1,95 @@
+// Copyright 2024 Canonical.
+package db_test
+
+import (
+	"context"
+	"database/sql"
+
+	qt "github.com/frankban/quicktest"
+	"github.com/juju/juju/state"
+
+	"github.com/canonical/jimm/v3/internal/db"
+	"github.com/canonical/jimm/v3/internal/dbmodel"
+)
+
+func (s *dbSuite) SetupDB(c *qt.C) (dbmodel.Model, dbmodel.Controller, dbmodel.Cloud) {
```

Comment:
Do we need this on the suite? Also bit surprised it didn't exist already

Reply:
there is but in other pkgs, I don't think we necessarly need this in the suite because it's useful for this specific use case.

Reply:
Then off suite it goes aha

Reply:
fair enough, lol